openedx/features/idea/api_views.py

```diff
@@ -0,0 +1,26 @@
+from django.http import JsonResponse
+from rest_framework import status
+from rest_framework.authentication import SessionAuthentication, BasicAuthentication
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.views import APIView
+
+from openedx.features.idea.models import Idea
+
+
+class FavoriteAPIView(APIView):
+    """
+    FavoriteAPIView is used t toggle favorite idea for the user
+    """
+    authentication_classes = (SessionAuthentication, BasicAuthentication)
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request, idea_id):
+        response = dict(message='User is added to favorites')
```

Comment:
Is user favoriting idea or user ?

Shouldn't this message be 'Idea is added to your favorites' similarly 'Idea is removed from your favorites'

Reply:
idea

Reply:
But message says otherwise